lm_diagnoser/extractors/base_extractor.py

```diff
@@ -41,7 +53,7 @@ class Extractor:
         corpus containing the labels for each sentence.
     hidden_size : int
         Number of hidden units in model.
-    activation_names : List[ActivationName]
+    activation : List[ActivationName]
```

Comment:
I renamed it because otherwise it would require a change of name in the config, in which it is callede 'activations' (because the config names now should match exactly the class names). Shall I change it in configs too?

Reply:
Ah you're right, that was sloppy on my end. Yes, I think that's best :+1:

Reply:
That's just what happens when you are coding, and why it is always good to have someone else code-review :). In general 'm surprised about how little "sloppy" things I found!

Reply:
Done